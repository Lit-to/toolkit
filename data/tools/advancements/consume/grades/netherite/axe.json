{
    "criteria": {
        "use_item": {
            "trigger": "minecraft:consume_item",
            "conditions": {
                "item": {
                    "items": [
                        "minecraft:netherite_axe"
                    ]
                }
            }
        }
    },
    "rewards": {
        "function": "tools:from_adv/grades/netherite/axe"
    }
}