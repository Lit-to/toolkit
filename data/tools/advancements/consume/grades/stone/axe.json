{
    "criteria": {
        "use_item": {
            "trigger": "minecraft:consume_item",
            "conditions": {
                "item": {
                    "items": [
                        "minecraft:stone_axe"
                    ]
                }
            }
        }
    },
    "rewards": {
        "function": "tools:from_adv/grades/stone/axe"
    }
}