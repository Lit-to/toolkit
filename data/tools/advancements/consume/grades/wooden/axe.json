{
    "criteria": {
        "use_item": {
            "trigger": "minecraft:consume_item",
            "conditions": {
                "item": {
                    "items": [
                        "minecraft:wooden_axe"
                    ]
                }
            }
        }
    },
    "rewards": {
        "function": "tools:from_adv/grades/wooden/axe"
    }
}