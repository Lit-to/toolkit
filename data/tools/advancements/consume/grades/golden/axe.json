{
    "criteria": {
        "use_item": {
            "trigger": "minecraft:consume_item",
            "conditions": {
                "item": {
                    "items": [
                        "minecraft:golden_axe"
                    ]
                }
            }
        }
    },
    "rewards": {
        "function": "tools:from_adv/grades/golden/axe"
    }
}